src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -20,10 +21,10 @@
 public class ArgsParserTest {
 
     private static final Path PROJECT_DIRECTORY = Paths.get(System.getProperty("user.dir"));
-    private static final Path CONFIG_FILE_ABSOLUTE = Paths.get(ArgsParserTest.class.getClassLoader()
-            .getResource("output/parser_test.csv").getPath());
-    private static final Path OUTPUT_DIRECTORY_ABSOLUTE = Paths.get(ArgsParserTest.class.getClassLoader()
-            .getResource("output").getPath());
+    private static final Path CONFIG_FILE_ABSOLUTE = Paths.get(new File(ArgsParserTest.class.getClassLoader()
+            .getResource("output/parser_test.csv").getPath()).getPath());
+    private static final Path OUTPUT_DIRECTORY_ABSOLUTE = Paths.get(new File(ArgsParserTest.class.getClassLoader()
+            .getResource("output").getPath()).getPath());
```

Comment:
Looks abit weird to have 2 `.getPath()` though 😅
Maybe this would look nicer and simpler?
```Java
new File(ArgsParserTest.class.getClassLoader()
            .getResource("output/parser_test.csv").getFile()).toPath();
```